lldb/test/API/functionalities/reverse-execution/TestReverseContinueBreakpoints.py

```diff
@@ -0,0 +1,119 @@
+import lldb
+import time
+import unittest
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+from lldbsuite.test.gdbclientutils import *
+from lldbsuite.test.lldbreverse import ReverseTestBase
+from lldbsuite.test import lldbutil
+
+
+class TestReverseContinueBreakpoints(ReverseTestBase):
+    @add_test_categories(["dwarf"])
```

Comment:
You probably want to set `NO_DEBUG_INFO_TESTCASE = True` class property instead.

Reply:
Yes, thanks for that.